gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,258 @@
+            Please refer to <NxTextLink href="#/pages/Modal">the Accessibility Section</NxTextLink> of the{' '}
+            <NxCode>NxModal</NxCode> documentation.
+          </NxP>
+        </NxTile.Subsection>
+      </GalleryDescriptionTile>
+
+      <NxTile>
+        <NxTile.Header>
+          <NxTile.HeaderTitle>
+            <NxH2>Simple NxDrawer Example</NxH2>
+          </NxTile.HeaderTitle>
+        </NxTile.Header>
+        <NxTile.Content>
+          <NxP>
+            A basic example of an <NxCode>NxDrawer</NxCode> with and without a footer.
+            This example also shows a NxDrawer with overflowing content.
+          </NxP>
+          <NxP>
+            <NxTextLink href="#/NxDrawerExample">
+              Click here to navigate to the live example.
+            </NxTextLink>
+          </NxP>
+          <CodeExample content={NxDrawerSourceCode}/>
+        </NxTile.Content>
+      </NxTile>
+
+      <NxTile>
+        <NxTile.Header>
+          <NxTile.HeaderTitle>
+            <NxH2>NxDrawer With Subtitle or Description Example</NxH2>
+          </NxTile.HeaderTitle>
+        </NxTile.Header>
+        <NxTile.Content>
+          <NxP>
+            An example of an <NxCode>NxDrawer</NxCode> with subtitle or description in the header.
+          </NxP>
+          <NxP>
+            <NxTextLink href="#/NxDrawerWithSubtitleOrDescriptionExample">
+              Click here to navigate to the live example.
+            </NxTextLink>
+          </NxP>
+          <CodeExample content={NxDrawerWithSubtitleOrDescriptionSourceCode}/>
+        </NxTile.Content>
+      </NxTile>
+
+      <NxTile>
+        <NxTile.Header>
+          <NxTile.HeaderTitle>
+            <NxH2>NxDrawer Variant Example</NxH2>
+          </NxTile.HeaderTitle>
+        </NxTile.Header>
+        <NxTile.Content>
+          <NxP>
+            An example of a <NxCode>NxDrawer</NxCode> with different variants.
+          </NxP>
+          <NxP>
+            <NxTextLink href="#/NxDrawerVariantExample">
+              Click here to navigate to the live example.
+            </NxTextLink>
+          </NxP>
+          <CodeExample content={NxDrawerVariantSourceCode}/>
+        </NxTile.Content>
+      </NxTile>
+
+      <NxTile>
+        <NxTile.Header>
+          <NxTile.HeaderTitle>
+            <NxH2>NxDrawer With NxForm Example</NxH2>
+          </NxTile.HeaderTitle>
+        </NxTile.Header>
+        <NxTile.Content>
+          <NxP>
+            An example of a <NxCode>NxDrawer</NxCode> with <NxCode>NxForm</NxCode>. This example
+            also shows <NxCode>NxDrawer</NxCode> with overflowing form content.
+          </NxP>
+          <NxP>
+            <NxTextLink href="#/NxDrawerWithNxFormExample">
+              Click here to navigate to the live example.
+            </NxTextLink>
+          </NxP>
+          <CodeExample content={NxDrawerWithNxFormSourceCode}/>
```

Comment:
It would be much better if these code examples where collapsed in accordions as is typical.  See the `NxToast` PR for how to do that.

Reply:
> It would be much better if these code examples were collapsed in accordions as is typical. See the `NxToast` PR for how to do that.

Fixed here: 8f98a6cc7e9a9d8dc6c1808db2df676e4da86a01